src/popover/popover.scss

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+
+// tippy.js helper class to hide Popover when it is scrolled out of view
+@mixin iui-popover {
```

Comment:
So where is this going to be applied? On the same level as `.tippy-content`?

We were talking about also resetting background-color and other stuff, which goes on the parent (`.tippy-box`).
![image](https://user-images.githubusercontent.com/9084735/152377018-274cdb2e-3d0b-4137-a6fd-44842a41f520.png)

Reply:
But we also noticed it was not the case in our storybook...? Did we figure out why its different?

Reply:
If users already have `tippy.js` dependency and also imported it's CSS, then it will take default styling:
![image](https://user-images.githubusercontent.com/36186912/152383121-ded0af7b-6708-41fa-8748-7ad7626914d8.png)

Reply:
hmm.. If it has background only when users directly import tippy css, I guess we should not override it, imo

Reply:
It won't override their tippy.js components as we are only targeting our tippy that will have `iui-popover` class.

Reply:
OK, then add it (check to be correct DOM structure)
I just wanted to save time.. :DD

Reply:
Added and tested.